examples/basic-js/package.json

```diff
@@ -5,10 +5,10 @@
     "test": "tests"
   },
   "dependencies": {
-    "@playwright/test": "0.162.0"
+    "@playwright/test": "0.1101.0"
   },
   "scripts": {
-    "test": "folio"
+    "test": "folio --config=config.js"
```

Comment:
I would call all the configs `folio.config.ts` then you don't have to specify it manually with the --config parameter

Reply:
Ok, I'll fix the examples :)